cfgov/hmda/templates/hmda/orange-explorer.html

```diff
@@ -740,8 +740,32 @@ <h1>Explore the data</h1>
 
       <div class="cf-error" id="msa-note">
         <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1200" class="cf-icon-svg"><path d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm-49.7 234.6c0-27.6 22.4-50 50-50s50 22.4 50 50v328.6c0 27.6-22.4 50-50 50s-50-22.4-50-50V339.8zm50 582.5c-39.6 0-71.7-32.1-71.7-71.7s32.1-71.7 71.7-71.7S572 811 572 850.6s-32.1 71.7-71.7 71.7z"></path></svg>
-        <p class="message">This tool will sunset in the coming months.</p>
-        <p><a href="/data-research/hmda-api-retirement/">Learn more about what to expect</a></p>
+        <p class="message">This tool will sunset on 9/25/2020.</p>
+        <p>
+          <a href="/data-research/hmda/api-retirement/">Learn more</a>.
```

Comment:
Is this page going to be updated to also include information about the retirement date?

Reply:
@Scotchester: Yep, the API retirement page is changing, too. You can see what that will look like in the preview environment for this PR right now, and I'm going to stage those changes on content in a bit.